tests/io/test.xpi.js

```diff
@@ -610,14 +610,14 @@ describe('Xpi.getFilesByExt()', function() {
 
     return myXpi.getFilesByExt('.html', '.htm')
       .then((htmlFiles) => {
-        assert.equal(htmlFiles.length, 3);
-        assert.equal(htmlFiles[0], 'index.html');
-        assert.equal(htmlFiles[1], 'second.htm');
-        assert.equal(htmlFiles[2], 'third.html');
+        expect(htmlFiles.length).toEqual(3);
+        expect(htmlFiles[0]).toEqual('index.html');
+        expect(htmlFiles[1]).toEqual('second.htm');
+        expect(htmlFiles[2]).toEqual('third.html');
 
         for (let i = 0; i < htmlFiles.length; i++) {
-          assert.ok(htmlFiles[i].endsWith('.html') ||
-                    htmlFiles[i].endsWith('.htm'));
+          expect(htmlFiles[i].endsWith('.html') ||
```

Comment:
This would be better to use https://facebook.github.io/jest/docs/expect.html#tomatchregexporstring